### ISSUE TITLE
ServiceNet Public view non-loggedin user URL Change

### ISSUE THREAD

**Issue by sidmundhra** [benetech/ServiceNet#1212]
ServiceNet Public View to be moved to servicenet.benetech.org. 

Alternatively, servicenet.benetech.org/public works if we can add a button to access the public view like 'Continue without logging in'.
It should look exactly like what is currently on the public view link i.e. https://servicenet.benetech.org/#/public and a search bar for searching purposes.

**Comment by johnhbenetech** [comment]
I vote for changing the home page to have something like 'Try Now' that takes you to the public view

**Comment by sidmundhra** [comment]
Yeah. That works too. As long as there is a CTA for a non-loggedin user to visit the public view.
